Issue 6430 - Fix build with bundled libdb

yselkowitz opened the pull request:
Description:
The libbdb_ro change (#6431) added a `WITH_LIBBDB_RO` automake conditional and a `db_bdb_srcdir` autoconf substitution which must also be defined when building --with-bundle-libdb.

Related: https://github.com/389ds/389-ds-base/pull/6431

/cc @vashirov

progier389 reviewed (approved):
LGTM

vashirov reviewed (approved):
LGTM!
FYI, author shows up as `From: rpm-build <rpm-build>`. If this is not expected, please update the commit.

yselkowitz committed:
Issue 6430 - Fix build with bundled libdb

Description:
The libbdb_ro change (#6431) added a `WITH_LIBBDB_RO` automake conditional
and a `db_bdb_srcdir` autoconf substitution which must also be defined when
building --with-bundle-libdb.

Related: https://github.com/389ds/389-ds-base/pull/6431
